portfolio/src/main/webapp/charts.js

```diff
@@ -17,22 +17,22 @@ google.charts.setOnLoadCallback(drawChart);
 
 /** Creates a chart and adds it to the page. */
 function drawChart() {
-  var data = google.visualization.arrayToDataTable([
-    ['Task', 'Hours per Day'],
-    ['Work',     11],
-    ['Eat',      2],
-    ['Commute',  2],
-    ['Watch TV', 2],
-    ['Sleep',    7]
-  ]);
+  fetch('/comments-authors-data')
+  .then(response => response.json())
+  .then((json) => {
```

Comment:
nit: fix indentation

```suggestion
  fetch('/comments-authors-data')
    .then(response => response.json())
    .then((json) => {
```